### ISSUE TITLE
Icon Request: fa-emacs

### ISSUE THREAD

**Issue by QuiGonJinn** [FortAwesome/Font-Awesome#7907]
I would like to request an emacs icon. Something rendered from ![emacs](http://cvs.savannah.gnu.org/viewvc/*checkout*/emacs/emacs/etc/images/icons/hicolor/16x16/apps/emacs.png)  or ![emacs](http://cvs.savannah.gnu.org/viewvc/*checkout*/emacs/emacs/etc/images/icons/hicolor/32x32/apps/emacs.png) . Thank you.

**Comment by HiCodin** [comment]
i would like to have this as well. +1

**Comment by ekadhanda** [comment]
i would like to have this as well. +1

**Comment by lepinkainen** [comment]
+1 from me too

**Comment by zerOnepal** [comment]
any updates on this
![please](https://media.giphy.com/media/CT5Ye7uVJLFtu/giphy.gif)

**Comment by winpat** [comment]
+1

**Comment by yyadavalli** [comment]
+1

**Comment by IvanMalison** [comment]
Has anyone made an attempt to do this?

**Comment by zerOnepal** [comment]
@IvanMalison i wish there exists quick/easy  "how to " guideline for this... so i could try :grin:

**Comment by IvanMalison** [comment]
@zerOnepal Yeah, I'd spend the time to do it if there were. Even cooler would be a a generator that could make one from any image.

**Comment by nullx002** [comment]
+1

**Comment by lbrayner** [comment]
+1 Emacs
+1 Vim

**Comment by javanux** [comment]
+1 Emacs

**Comment by mohamed-aziz** [comment]
+1

**Comment by jcllompart** [comment]
+1

**Comment by eailfly** [comment]
+1

**Comment by jcllompart** [comment]
I've just added Emacs icon to my i3 bar from the Atom file-icon.

**Comment by thapakazi** [comment]
@jcllompart you have any breadcrumbs left for us signaling how you did that ?

**Comment by jcllompart** [comment]
goto
https://github.com/file-icons/atom/tree/master/fonts
and grab file-icons.woff2
then transform woff2 to ttf with any online tools -i did it with https://everythingfonts.com/woff2-to-ttf-
download the ttf file and install new fonts -i did it this way:
$ sudo mv file-icons.ttf /usr/share/fonts
$ sudo fc-cache -fv   // or just restart your computer
then goto
http://bluejamesbond.github.io/CharacterMap/
and select your file-icons.ttf file.
then just search for the emacs icon, select, and copy it from the Character item.
you just paste it whenever you want in your i3 config file.
![i3baremacs](https://cloud.githubusercontent.com/assets/21229029/24995978/4a9d964c-2031-11e7-8858-8e54e76eb856.png)
That's how it looks now.

**Comment by jcllompart** [comment]
although i don't usually have the i3bar visible on my desktop.

**Comment by thapakazi** [comment]
that's very verbose... **thanks for sharing** :bowing_man:  @jcllompart 

the wait for awesome icons is no more a advent.
![i3_revamped_1](https://cloud.githubusercontent.com/assets/1695906/25000157/f41c4988-205f-11e7-9f61-04c0837908c1.png)

**Comment by LeandroCardoso** [comment]
+1

**Comment by IvanMalison** [comment]
Shouldn't we use the new emacs icon which dropped the pen from the log?

**Comment by dikeert** [comment]
+1

**Comment by joehillen** [comment]
@dikeert Don't just say "+1"! It notifies everyone in the thread!

**Comment by azzamsa** [comment]
Instead of taking additional step to convert `file-icons.woff2` to ttf. Just use ttf fonts provided by [all-the-icons.el](https://github.com/domtronn/all-the-icons.el/tree/master/fonts)

![image](https://user-images.githubusercontent.com/17734314/54065729-97548f00-4257-11e9-83d5-c68c5f0110c6.png)

**Comment by Andryas** [comment]
+1

**Comment by avronr** [comment]
It would be better to have the updated icon.

[Emacs Icon](https://upload.wikimedia.org/wikipedia/commons/thumb/0/08/EmacsIcon.svg/1200px-EmacsIcon.svg.png)
![Emacs Icon](https://upload.wikimedia.org/wikipedia/commons/thumb/0/08/EmacsIcon.svg/1200px-EmacsIcon.svg.png "Emacs Icon")

**Comment by kguidonimartins** [comment]
+1

**Comment by manikandanraji** [comment]
> Hi, thanks for your interest in this request
> 
> Please use reactions to upvote, using comments is against our contributing guidelines

+1

**Comment by nullx002** [comment]
yes, is it there or not?

**Comment by p510n1c** [comment]
+1

**Comment by j-the-ripper** [comment]
its not closed since 2015 ;-;

**Comment by someonewithpc** [comment]
+1

**Comment by alinelena** [comment]
+2

**Comment by rdfmath** [comment]
I think maybe  it's time to give up hope.   What are people using instead of FontAwesome?

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
